index.html

```diff
@@ -33,9 +33,9 @@
     </section> <!-- end section 1 -->
 
     <!-- second section (headline) -->
-    <section>
+    <section class="section2">
       <main class="headline">
-        <div>
+        <div class="bg-image">
           <img src="images/bg.png" alt="burger-menu" class="bg-image">
         </div>
```

Comment:
- [REQUIRED] Kindly, the background image always backgrounds images, not `img` tag, use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.👌👌Please, Remove the `<img>` tag and use it in CSS `background-image` as your previous code reviewer already suggested you.👌👌